index.html

```diff
@@ -1,192 +1,192 @@
-                    <img class="dot" src="img/dot.png" alt="dot">
-                    <a href="#" class="tech">BACK END DEV</a>
-                    <img class="dot" src="img/dot.png" alt="dot">
-                    <a href="#" class="year">2015</a>
+            </article>
+            <article class="box">
+                <img class="image" src="img/tonic_2.png" alt="tonic">
+                <div class="box-info">
+                    <h2 class="title">Tonic</h2>
+                    <div class="description">
+                        <a href="#" class="client">CANOPY</a>
+                        <img class="dot" src="img/dot.png" alt="dot">
+                        <a href="#" class="tech">BACK END DEV</a>
+                        <img class="dot" src="img/dot.png" alt="dot">
+                        <a href="#" class="year">2015</a>
+                    </div>
+                    <p class="details">A daily selection Of privately personalized reads; no accounts or sign-ups required
+                    </p>
+                    <ul class="stack">
+                        <li class="style">html</li>
+                        <li class="style">css</li>
+                        <li class="style">javascript</li>
+                    </ul>
+                    <a href="#" class="preview">See Project</a>
                 </div>
-                <p class="details">A daily selection Of privately personalized reads; no accounts or sign-ups required
-                </p>
-                <ul class="stack">
-                    <li class="style">html</li>
-                    <li class="style">css</li>
-                    <li class="style">javascript</li>
-                </ul>
-                <a href="#" class="preview">See Project</a>
-            </div>
-        </article>
-        <article class="box">
-            <img class="image" src="img/multi_2.png" alt="multi">
-            <div class="box-info">
-                <h2 class="title">Multi-Post Stories</h2>
-                <div class="description">
-                    <a href="#" class="client">CANOPY</a>
-                    <img class="dot" src="img/dot.png" alt="dot">
-                    <a href="#" class="tech">BACK END DEV</a>
-                    <img class="dot" src="img/dot.png" alt="dot">
-                    <a href="#" class="year">2015</a>
+            </article>
+            <article class="box">
+                <img class="image" src="img/multi_2.png" alt="multi">
+                <div class="box-info">
+                    <h2 class="title">Multi-Post Stories</h2>
+                    <div class="description">
+                        <a href="#" class="client">CANOPY</a>
+                        <img class="dot" src="img/dot.png" alt="dot">
+                        <a href="#" class="tech">BACK END DEV</a>
+                        <img class="dot" src="img/dot.png" alt="dot">
+                        <a href="#" class="year">2015</a>
```

Comment:
- [ ] :clap: Creativity at its peak! I will only suggest that since this is more like a list of items with similar attributes, you can use a `ul`/`li` tag to enclose the `a` tags and the `img` tags. How about that :grey_question: